src/utils/const.js

```diff
@@ -0,0 +1,44 @@
+const borderWidth = 0;
```

Comment:
Minor Opinion:
how about use plural form like 'constants' or 'const_values' for filename instead a 'const'.